cmd/version.go

```diff
@@ -26,7 +26,7 @@ const unknown = "unknown"
 // var needs to be used instead of const as ldflags is used to fill this
 // information in the release process
 var (
-	kubeBuilderVersion      = unknown
+	cliVersion              = unknown
```

Comment:
I think here we should keep it as before using `kubeBuilderVersion`
It is not used for external projects.
Only in the places that are imported and used for external projects we need use generic terms and names